src/remarx/quotation/embeddings.py

```diff
@@ -0,0 +1,37 @@
+"""
+Provides functionality to generate sentence embeddings from sentence corpora
+using pretrained models from the sentence-transformers library.
+"""
```

Comment:
Seems like we could probably reduce this to something like this
```suggestion
Library for generating sentence embeddings from pretrained Sentence Transformer models.
```